server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityAnalysis.js

```diff
@@ -65,7 +65,7 @@ export default class ProjectActivityAnalysis extends React.PureComponent {
         role="listitem"
         tabIndex="0">
         <div className="project-activity-time spacer-right">
-          <FormattedDate className="text-middle" date={date} format="LT" tooltipFormat="LTS" />
+          <TimeTooltipFormatter className="text-middle" date={date} />
```

Comment:
this doesn't fit in one one:

![image](https://user-images.githubusercontent.com/2317341/29558687-2c2c2062-872d-11e7-89d1-c1a1986e58f9.png)